docs/reference/api-gateway/http.md

```diff
@@ -0,0 +1,171 @@
+# API Gateway with http requests
```

Comment:
there's only apigateway for http requests, so perhaps no need to explicitly specify that